doc/source/whatsnew/v1.1.0.rst

```diff
@@ -339,6 +339,10 @@ Backwards incompatible API changes
 - Combining a ``Categorical`` with integer categories and which contains missing values
   with a float dtype column in operations such as :func:`concat` or :meth:`~DataFrame.append`
   will now result in a float column instead of an object dtyped column (:issue:`33607`)
+- :meth:`Series.to_timestamp` now raises a  ``TypeError`` if the axis is not a :class:`PeriodIndex`.
+  Previously an ``AttributeError`` was raised (:issue:`33327`)
```

Comment:
can you put this on the previous line

Reply:
Did you mean to add it as single line.
Modified it as single line.
Also, other changes updated as suggested.